javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,62 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
```

Comment:
These console.logs are fine when learning, but it's unusual to have them on actual production sites.